regenerate.sh

```diff
@@ -29,7 +29,10 @@ mkdir -p ${GOBIN}
 echo "remove existing generated files"
 # grpc_testing_not_regenerate/*.pb.go is not re-generated,
 # see grpc_testing_not_regenerate/README.md for details.
-rm -f $(find . -name '*.pb.go' | grep -v 'grpc_testing_not_regenerate')
+#
+# Remove generated pb.go files in all directories with addition of
+# excluding grpc_testing_not_regenerate and cmd/protoc-gen-go-grpc directories
+rm -f $(find . -name '*.pb.go' | grep -v -e 'grpc_testing_not_regenerate' -e 'cmd/protoc-gen-go-grpc')
```

Comment:
Actually, I have renamed the files in `grpc_test_not_regenerate` to not be `.pb.go` anymore, so you can delete this part.

Reply:
will do